java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java

```diff
@@ -110,17 +116,36 @@ public ResponseEntity<Object> invite(
         }
     }
 
+    @NotNull
+    private static ResponseParams createSchemaNotFoundResponse(String entityName, ResponseParams responseParams) {
+        ResponseParams responseParams1;
```

Comment:
use the passed responseParams instead of creating a new object of responseParams